### DISCUSSION TITLE
StrictTransportSecurity header excluded

### DISCUSSION THREAD

**Discussion by personwithproxy** [dotnet/yarp#2288] [Q&A] [answered]
The StrictTransportSecurity header is excluded (see RequestUtilities.cs) by default.
1. To enable it, is there a better way than to build my own Nuget package?
2. Some security scanner mark the absence of the header as security risk. What's the reason to exclude this in YARP by default?

Using version 2.0.1.
Thanks!

**Comment by Tratcher** [comment, marked as answer]
See https://github.com/microsoft/reverse-proxy/issues/2269
